package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "api-snippets",
+  "version": "0.0.1",
+  "description": "Mojaloop API specification reusable snippets",
+  "main": "resolve.js",
+  "scripts": {
+    "start": "node resolve.js",
+    "test": "node resolve.js index.yaml > test/resolved.json && node resolve.js -o yaml index.yaml > test/resolved.yaml && node test/diffCheck.js",
+    "buildv1.0": "node resolve.js -o yaml v1.0/openapi3/index.yaml > docs/fspiop-rest-v1.0-openapi3-snippets.yaml"
+  },
+  "bin": {
+    "multi-file-swagger": "resolve.js"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "commander": "^2.19.0",
+    "js-yaml": "^3.12.2",
+    "json-refs": "^3.0.12"
+  },
+  "devDependencies": {
```

Comment:
where is linting support to keep .js files in sync with standard?

Reply:
Curse my piece by piece mentality. I'll fit that into this PR.